docs/development.md

```diff
@@ -1,15 +1,61 @@
-Development of Unfolded plugin
-===========================
+Plugin development
+==================
 
-This project uses [qgis_plugin_tools](https://github.com/GispoCoding/qgis_plugin_tools) submodule, so when cloning
-use `--recurse-submodules` like so:
-`git clone --recurse-submodules https://github.com/UnfoldedInc/qgis-plugin.git`
+## Setup
 
-The code for the plugin is in the [Unfolded](../Unfolded) folder. Make sure you have required tools, such as Qt with Qt
-Editor and Qt Linquist installed by following this
-[tutorial](https://www.qgistutorials.com/en/docs/3/building_a_python_plugin.html#get-the-tools).
+1. Install QGIS app from https://qgis.org/en/site/forusers/download.html
+2. We rely on [qgis_plugin_tools](https://github.com/GispoCoding/qgis_plugin_tools), so when cloning the repo, make sure to clone it recursively, with submodules:
 
-For building the plugin use platform independent [build.py](../Unfolded/build.py) script.
+```bash
+git clone --recurse-submodules https://github.com/UnfoldedInc/qgis-plugin.git
```

Comment:
I think this URL has changed to foursquare github org (the old URL probably just redirects)

Reply:
I left renames for a different PR (concretely #63). Also, for now, Github still handles leftover redirects from all old org (UnfoldInc) to the new one (foursquare).